configuration.md

```diff
@@ -14,10 +14,12 @@ Casual supports the following configuration options which can be set using envir
 * `CASUAL_CONFIG_FILE` - Set path for casual configuration file. If not provided, default are used.
 * `CASUAL_INBOUND_STARTUP_MODE` - Set mode for inbound startup. Default `immediate`. Alternatives `trigger`, `discover`.
     See [Inbound Startup Configuration](inbound.md#startup-configuration) for more details.
-* `CASUAL_OUTBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
-* `CASUAL_INBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
+* `CASUAL_OUTBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO - deprecated, please see `CASUAL_OUTBOUND_USE_EPOLL` instead.
```

Comment:
This documentation says deprecated, but from what I can see the use of this environment variable has been outright removed. Does anyone currently use this env who will be affected by the change and switch?

Reply:
Hmm, did I remove it by accident?
I'll add it back.